serviceWorker.js

```diff
@@ -0,0 +1,214 @@
+//Cache polyfil to support cacheAPI in all browsers
+importScripts('./cache-polyfill.js');
+
+var cacheName = 'cache-v4';
```

Comment:
I don't quite understand what naming the cache is good for. Is this just going to stay consistent?

Reply:
Well Its for offline , workflow of the app!